src/main/java/com/iridium/iridiumskyblock/commands/RegenCommand.java

```diff
@@ -47,7 +47,7 @@ public boolean execute(User user, Island island, String[] args, IridiumTeams<Isl
 
         Schematics.SchematicConfig schematicConfig = IridiumSkyblock.getInstance().getSchematics().schematics.get(schematic.get());
 
-        if(IridiumSkyblock.getInstance().getConfiguration().payPerRegen) {
+        if(schematicConfig.regenCost.money == 0 && schematicConfig.regenCost.bankItems.isEmpty()) {
```

Comment:
This would be if its not 0, and its not empty right?

Reply:
uh yeah, that should be right
this is why i tend to have too many parentheses, i prefer to write them like math equations 
ex:
```java
if((schematicConfig.regenCost.money == 0) && (schematicConfig.regenCost.bankItems.isEmpty()))
```